network/dns2proxy/Makefile

```diff
@@ -0,0 +1,51 @@
+
+#
+# Copyright (C) 2011 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=dns2proxy
+PKG_VERSION:=1.0
+PKG_RELEASE:=3
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/LeonardoNve/dns2proxy.git
```

Comment:
Please use https://codeload.github.com/x for the package sources.

Reply:
Some git repos didn't tad the releases.
So i've edited those Makefiles to use the source locally within a subdir.